hangups/conversation.py

```diff
@@ -473,6 +473,55 @@ async def send_message(self, segments, image_file=None, image_id=None,
                 logger.warning('Failed to send message: {}'.format(e))
                 raise
 
+    async def add_user(self, *user_ids):
+        """Add one or more users to this conversation.
+
+        Args:
+            user_ids (str...): IDs of the new users.
+
+        Raises:
+            TypeError: If conversation is not a group.
+            NetworkError: If conversation cannot be invited to.
+        """
+        if not self._conversation.type == hangouts_pb2.CONVERSATION_TYPE_GROUP:
+            raise TypeError('Conversation is not a group')
```

Comment:
Would it be better to add a new exception for this, like `ConversationTypeError`? `TypeError` or `ValueError` don't seem appropriate since the conversation isn't an argument to this method.